README.rst

```diff
@@ -1 +1,138 @@
-Stub readme.
+edx_event_bus_kafka
+=============================
+
+|pypi-badge| |ci-badge| |codecov-badge| |doc-badge| |pyversions-badge|
+|license-badge|
+
+The ``README.rst`` file should start with a brief description of the repository,
+which sets it in the context of other repositories under the ``edx``
+organization. It should make clear where this fits in to the overall edX
+codebase.
+
+Kafka implementation for Open edX event bus.
```

Comment:
Author note: I think this should be moved above the placeholder paragraph in the cookiecutter.

Reply:
Yeah, I'd put it just below the title